apps/dotcom-worker/src/lib/routes/createRoomSnapshot.ts

```diff
@@ -18,7 +23,6 @@ export async function createRoomSnapshot(request: IRequest, env: Environment): P
 
```

Comment:
maybe we could include the parent slug as a prefix for the room id

```ts
const roomId = `${data.parent_slug}/v2_c_${nanoid()}`
```

that way we can list/delete all snapshots associated with a particular room if we need to.

not 100% sure a non-url-safe char (`/`) will be ok when including the id in urls but 🤞🏼

Reply:
We probably wouldn't want to expose the parent slug though, otherwise people can get back to the original? Some rooms also don't have a parent slug - you can create a snapshot from your local room.

One thing we could do is store the parent slug in a KV, similar to what we do for readonly rooms. Then we could store snapshots in the R2 under the following paths:

* `public_rooms/parent_slug/snapshot_slug` for the snapshots that do have a parent
* `public_rooms/snapshot_slug` for the snapshots that don't have it

Reply:
Added the above suggestion.

Reply:
ah cool. yeah I forgot about the privacy issue. kv works well!